bases/renku_data_services/storage_api/app.py

```diff
@@ -76,6 +78,10 @@ def put(self) -> BlueprintFactoryResponse:
         @authenticate(self.authenticator)
         @only_admins
         async def _put(request: Request, storage_id: str, validator: RCloneValidator, user: base_models.User):
+            if not request.json:
+                raise errors.ValidationError(message="The request body is empty. Please provide a valid JSON object.")
+            if not isinstance(request.json, dict):
+                raise errors.ValidationError(message="The request body is not a valid JSON object.")
```

Comment:
The schemas for the api spec do not enforce this?

Reply:
`**request.json` fails in that case, hypothesis found this issue.

Reply:
And I can't `@validate()` since that doesn't work when there's multiple schemas. or at least I didn't find a way to do that